trie/hasher.go

```diff
@@ -50,19 +43,28 @@ func (b *sliceBuffer) Reset() {
 	*b = (*b)[:0]
 }
 
-// hashers live in a global db.
+// hasher is a type used for the trie Hash operation. A hasher has some
+// internal preallocated temp space
+type hasher struct {
+	sha keccakState
+
```

Comment:
Pls delete this empty line, really bothers me that fields are not aligned :P Espectially since they are different in the committer

Reply:
Yeah, and that `tmpKey` looks like a remnant of something I later did not use, and should be removed

Reply:
No, I guess this can be fully internal